include/deal.II/matrix_free/shape_info.templates.h

```diff
@@ -238,6 +297,169 @@ namespace internal
     {
       static_assert(dim == spacedim,
                     "Currently, only the case dim=spacedim is implemented");
+
+      // ShapeInfo for RT elements. Here, data is of size 2 instead of 1.
+      // data[0] is univariate_shape_data in normal direction and
+      // data[1] is univariate_shape_data in tangential direction
+      //
+      if (dynamic_cast<const FE_RaviartThomasNodal<dim> *>(
+            &fe_in.base_element(base_element_number)))
+        {
+          element_type = tensor_raviart_thomas;
+
+          const auto quad = quad_in.get_tensor_basis()[0];
+
+          const FiniteElement<dim> &fe =
+            fe_in.base_element(base_element_number);
+          n_dimensions = dim;
+          n_components = fe_in.n_components();
+
+          data.resize(2);
+
+          // data_access.reinit(n_dimensions, n_components); // what are these?
+          // data_access.fill(&univariate_shape_data);
+
+          if ((fe.n_dofs_per_cell() == 0) || (quad.size() == 0))
+            return; // When do we want to check this?
```

Comment:
I don't think we need this code. The first case can never happen as an RT element has at least 2*dim unknowns (values on faces), whereas the second case would be an empty quadrature formula.